torch/csrc/jit/frontend/schema_emitter.cpp

```diff
@@ -0,0 +1,144 @@
+#include <torch/csrc/jit/frontend/schema_emitter.h>
```

Comment:
I agree. I had a `TODO` in `schema_emitter.h`, but that was a bit lazy...I've since been able to set everything up so that this logic can be in `ir_emitter_utils.h`